docs/reference/tags/form_tags/clear_cart/index.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: default
+title: Clear Cart
+parent: Tags
+has_children: false
+---
+
+The Clear Cart Tag renders a button that clears the user's cart. 
+
+##### input
+{% raw %}
+```liquid
+{% form "clear_cart" %}
+	 <input type="submit" value="clear" />
+ {% endform %}
+```
+{% endraw %}
+
+##### output
+{% raw %}
+```html
+<form action="/sites/cart" accept-charset="UTF-8" method="post">
+    <input type="hidden" name="_method" value="delete" autocomplete="off">
+    <input type="hidden" name="return_to" id="return_to" value="https://www.creator-website.com/book-tickets" autocomplete="off">
+    <input type="submit" value="Clear">
+</form>
+```
+{% endraw %}
+
+##### redirect
+The Clear Cart Tag will redirect to the cart page by default.
```

Comment:
I think by default it will reload the user's current page

Reply:
Thanks, I'll update that now!